### ISSUE TITLE
Some Feedback 

### ISSUE THREAD

**Issue by chaseadamsio** [markos84uk/potential-train-wreck#1]
:wave: really awesome work, it's pretty amazing to see how fast you're movin! here are some things I noticed:

- **Consider adding a README.md.** Usually they answer questions like "what does this do?", "What do I need installed to run it?", "how do I contribute to this thing?", "How do I start it up locally?". It's not _technical_ feedback, but it is one of the first things I do when I create a new project. This is _super useful_ for future Mark who hasn't touched this in a while and helpful for when people pull your project. It also helps you know you understand it because you wrote what's happening in your own words: https://github.com/chaseadamsio/chaseonsoftware.com#development
- *Ignore `node_modules`.* I see you added `node_modules` to your `.gitignore` but you already had them in your git history so they're in version control. As an exercise, how can you remove those from version control and make sure they don't wind up back in git? 
- *Tricky dev question.* What would happen for me as a developer with a new computer and I pull down your project and I have node installed and run `npm run watch`? What can you do to prevent what happens currently (it'll be an error) from being a problem in the future?
- *move all of your secrets to environment variables (and out of version control).* Environment variables are a good way to share secrets without them ever being in a repo. In this case you have two things that look like secrets: your mongo credentials on this line https://github.com/markos84uk/potential-train-wreck/blob/master/server.js#L16 and your base64 encoded authorization header. I believe Heroku has a way to setup secure keys and that's what you'd want to use here.
- *move inline files served to static filesystem files.* as your app grows, it's important to move all of your inline files that are served to filesystem files and import them. Moving that will help you out a lot and you can probably leverage your existing public directory.

If you have any questions, feel free to reach out!

**Comment by markos84uk** [comment]
This is definitely how I'm moving fast at the moment 😂 

![giphy (55)](https://user-images.githubusercontent.com/43969188/65780815-7182cf00-e142-11e9-8234-28bc07c33684.gif)

Thank you again so much for this feedback @chaseadamsio 
It's really delivered in such a kind and helpful way, and some great things to keep me moving forward. PR opened and will re-ping for a review soon! 🙌 🙏 

PR #2

**Comment by markos84uk** [comment]
@chaseadamsio + Federico. I'm closing this project down, and building a new React app from scratch. I followed a tutorial for this one, and realised I followed so closely, that I haven't really learned anything. 

At least with the new project, it will be from what I have learned so far, and I will encounter bugs and learn how to solve them. I'll also use the most basic syntax, without syntactic sugar like axios, which is useful, but better used when I understand what's going on underneath.

Thank you for your feedback here, I will be applying it to the new project, and if you're happy for me to request reviews from you I would love to do so 😄 @chaseadamsio
